server/flaws.js

```diff
@@ -48,7 +48,7 @@ function anyMatchSearchFlaws(searchFlaws, flaws) {
 function validPopularityFilter(value) {
   let filter = null;
   if (value) {
-    if (/[^\d<>]/.test(value)) {
+    if (/[^\d<>]/.test(value.replace(/\s+/g, ""))) {
```

Comment:
THis is an unrelated thing but I noticed that if you try to filter by `< 100` on the popularity it would break. You could use `<100` which would have worked.